packages/terra-modal/src/Modal.jsx

```diff
@@ -97,13 +121,14 @@ class Modal extends React.Component {
           onOpen,
           onUpdate,
           role,
+          onRequestClose,
            ...customProps } = this.props;
 
     return (
       <Portal
         isOpened={isOpened}
-        closeOnEsc={closeOnEsc}
-        closeOnOutsideClick={closeOnOutsideClick}
+        closeOnEsc={false}
```

Comment:
Could we just remove these guys completely? Or do they default the values to true?